packages/react-core/src/components/Select/examples/Select.md

```diff
@@ -4,7 +4,7 @@ section: components
 subsection: menus
 cssPrefix: pf-v6-c-select
 propComponents:
-  ['Select', 'SelectOption', 'SelectGroup', 'SelectList', 'MenuToggle', 'SelectToggleProps', 'SelectPopperProps']
```

Comment:
Overall, I _think_ this isn't introducing a breaking change. but it's possible that these extra interfaces were pulled out the way they were so that they could be documented in our API docs on the website. People won't know without looking at the code how to manipulate these popper features like direction and width if we remove the interfaces all together.

It is possible we could include Popper props on the API docs instead. I'm interested what the team thinks

Reply:
I don't think we would want to include PopperProps since it includes some attributes that aren't in the extended types.

What's the motivation for pulling the extended types out?

Reply:
`SelectPopperProps` doesn't shows any of the properties inherited from `PopperProps`, so there is no point in showing it in the documentation

Reply:
Yeah I think we would need to still keep it around until our next breaking change